javascript/javascript3/week1/weather.js

```diff
@@ -0,0 +1,111 @@
+const main = document.querySelector('main');
+const cityInput = document.getElementById('city');
+const para = document.querySelector('p');
+const getData = document.getElementById('weather-btn');
+
+getData.addEventListener('click', getWeatherData);
+
+//Fire Event from Enter
+cityInput.addEventListener('keyup', event => {
+    if (event.key === 'Enter') {
+        getWeatherData (); 
+    }
+})
+// Reach API
+function getWeatherData (){
+    const cityName = cityInput.value.toLowerCase();
+    const api = "https://api.openweathermap.org/data/2.5/weather?q=";
+    const key = "&appid=0f5d1d4ac034115ef7ae1bcdcada3527";
```

Comment:
i would have `const key = "0f5d1d4ac034115ef7ae1bcdcada3527"`, and then compose the `url` variable with `&appid=${key}`. At least it's more logical that way.